applications/Preprocessor/output_impl.h

```diff
@@ -165,6 +177,8 @@ void outputMesh(Mesh<dimension>& mesh,
         }
         output << std::endl;
     }
+
+    output << "elements\n";
     for (auto element : mesh.getElements()) {
```

Comment:
I don't see the output of which element belongs to which zone here

Reply:
argh I forgot that part.... oops